### DISCUSSION TITLE
Question mark instead of path

### DISCUSSION THREAD

**Discussion by hazy77** [JanDeDobbeleer/oh-my-posh#4787] [Troubleshoot] [answered]
Hello,

Can you tell me why on my home path I can see question mark instead of home icon or the path name? It is happening only there. I've checked folder rights and all seems to be ok.
 ![wt](https://github.com/JanDeDobbeleer/oh-my-posh/assets/15308073/ab406a22-94e7-41c1-9b3e-f3b78c6ed1a3)

**Comment by JanDeDobbeleer** [comment, marked as answer]
Could it be you have an outdated nerd font?

**Reply by hazy77** [reply]
Hi, yes that was it. Thanks for hint. Own you a beer ;)
